dev/com.ibm.ws.jbatch.open_fat/publish/servers/checkpointbatchFAT/server.xml

```diff
@@ -1,51 +1,29 @@
 <server>
-    <featureManager>
-        <feature>servlet-3.1</feature>
-        <feature>batch-1.0</feature>
-        <feature>appSecurity-2.0</feature>
-		<feature>jpa-2.0</feature>
-		<feature>cdi-1.2</feature>
-    </featureManager>
-
-<!--
-    <application type="war" id="snoop" name="snoop" location="${shared.app.dir}/webcontainer/snoop" />
-    -->
-    <include location="../fatTestPorts.xml"/>
-    
-    <!-- NOW THIS MESSAGE CHANGED -->
-    
-     <!-- batch container persistence config -->
-     <!--
+
+    <include location="../fatTestPorts.xml" />
```

Comment:
You should delete the original file at `dev/com.ibm.ws.jbatch.open_fat/publish/files/CDITestCheckpoint/server.xml`